.github/workflows/dotnet-core.yml

```diff
@@ -18,12 +18,7 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 0 # fetching all
-
```

Comment:
Why remove this? It makes which version of dotnet you are using deterministic and if that version is already installed it knows it and doesn't do anything.